### PR TITLE
DOC-2230: add new `How to fix invalid API key in TinyMCE` page to `tinymce/docs` project.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2230

Changes:
* add new FAQ tree to tinymce-docs project that contains `Invalid API Key` page.
* small `typo` correction for tinymce 6.8.1 docs

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -529,4 +529,5 @@
 **** xref:6.0-release-notes-premium-changes.adoc[Premium changes]
 **** xref:6.0-release-notes-known-issues.adoc[Known issues]
 ** xref:changelog.adoc[Changelog]
+* xref:invalid-api-key.adoc[Invalid API key]
```
Comment (MitchC1999) [inline modules/ROOT/nav.adoc:532]: Shouldn't this be under cloud deployment?

File: modules/ROOT/pages/invalid-api-key.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
+= Invalid API key
```
Comment (MitchC1999) [inline modules/ROOT/pages/invalid-api-key.adoc:1]: I thought we were going to make a generic cloud FAQ page?

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Review by MitchC1999 — APPROVED** [review]
Hopefully we can find a better solution soon

**Pull request merged**